diff --git a/node_modules/react-native-job-queue/android/build.gradle b/node_modules/react-native-job-queue/android/build.gradle
index ce463bf..942dcd0 100644
--- a/node_modules/react-native-job-queue/android/build.gradle
+++ b/node_modules/react-native-job-queue/android/build.gradle
@@ -11,7 +11,7 @@ buildscript {
 }
 
 plugins {
-    id "com.diffplug.gradle.spotless" version "3.23.0"
+    id "com.diffplug.spotless" version "5.0.0"
 }
 
 spotless {
